org.lflang/src/org/lflang/generator/GeneratorBase.xtend

```diff
@@ -802,9 +820,10 @@ abstract class GeneratorBase extends AbstractLFValidator {
         FederateInstance receivingFed,
         InferredType type,
         boolean isPhysical,
-        Delay delay
+        Delay delay,
+        SupportedSerializers serializer
     ) {
-        throw new UnsupportedOperationException("This target does not support direct connections between federates.")
+        throw new UnsupportedOperationException("This target does not support network connections between federates.")
```

Comment:
This message is repeated quite a few times. Should we create an enum with frequent error messages?

Reply:
Yes I think that is a good idea.